### DISCUSSION TITLE
Mécanisme de troncature

### DISCUSSION THREAD

**Discussion by cmjcz** [publicodes/publicodes#367] [Ideas]
Hello !
J’étais en train d’expérimenter avec le bac à sable pour implémenter le calcul des APL (que j’ai déjà implémenté totalement sur un tableur)
Je suis néanmoins confrontée à un obstacle. À la toute fin du calcul, je dois faire un arrondi à l’entier inférieur. Hors, le mécanisme d’arrondi, si je saisis bien la doc, n’est pas en mesure de le faire.
Je me demandais donc si il était possible d’implémenter un mécanisme de troncature ?
Également, un moment dans le calcul, j’ai besoin d’arrondir à la centaine supérieure.
Je trouve que le mécanisme d’arrondi manque de flexibilité pour effectuer ces opérations

Bonne journée à vous ✨

**Comment by johangirod** [comment]
Une issue est créée à ce sujet : #527
